src/encoded/static/components/navigation/components/LoginSuccessView.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import _ from 'underscore';
+import * as JWT from '@hms-dbmi-bgm/shared-portal-components/es/components/util/json-web-token';
+import { event as trackEvent, setUserID } from '@hms-dbmi-bgm/shared-portal-components/es/components/util/analytics';
+import { load, fetch, promise as ajaxPromise } from '@hms-dbmi-bgm/shared-portal-components/es/components/util/ajax';
+import { Alerts } from '@hms-dbmi-bgm/shared-portal-components/es/components/ui/Alerts';
+import { navigate } from '@hms-dbmi-bgm/shared-portal-components/es/components/util';
+import { ItemDetailList } from '@hms-dbmi-bgm/shared-portal-components/es/components/ui/ItemDetailList';
+
+
+/**
+ * Component that essentially handles the "callback" interaction previously done by
+ * the UI. The interaction in the Redis state now proceeds as follows:
+ *      1. The UI renders the Auth0Lock component in 'code' mode
+ *      2. The user logs in via the Auth0Lock and transmits the code to the back-end 
+ *         via GET /callback?code=abcdefg...
+ *      3. The back-end calls into Auth0 to get JWT and returns a session token to
+ *         the browser, returning a success response
+ *      4. This component loads, triggering the acquisition of /session-properties
+ *         and thus the population of the user_info in local storage
+ */
+export default class LoginSuccessView extends React.PureComponent {
+
+    static propTypes = {
+        'readyToRedirect': PropTypes.bool
+    };
+    
+    constructor(props) {
+        super(props);
+        this.state = {
+            readyToRedirect: false
+        };
+    }
+
+    /**
+     * This component is meant to be loaded upon navigating the /callback
```

Comment:
suggestion: "upon navigating from the backend (on authentication) back to the /callback"